shopyo/api/tests/test_validators.py

```diff
@@ -0,0 +1,15 @@
+import pytest
+from shopyo.api import validators
+
+class TestValidators:
+    """Tests validator methods"""
+
+    @pytest.mark.parametrize("url,expected", [("google", False),
+    ("localhost:3000", True), ("192.168.0.250", True),
+    ("2001:0db8:85a3:0000:0000:8a2e:0370:7334", True)])
+    def test_is_valid_url(self, url, expected):
+        result = validators.is_valid_url(url)
+        if expected:
+            assert result != None 
+        else:
+            assert result == None 
```

Comment:
something like this might be better for readability.
```python
    @pytest.mark.parametrize("url,expected", [("google", False),
    ("localhost:3000", True), ("192.168.0.250", True),
    ("2001:0db8:85a3:0000:0000:8a2e:0370:7334", True)])
    def test_is_valid_url(self, url, expected):
        result = bool(validators.is_valid_url(url))

        assert result == expected
```
But if you change the return type of `is_valid_url` to `True/False` then it you can remove the `bool(` from above